google-cloud-translate/pom.xml

```diff
@@ -2,6 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>google-cloud-translate</artifactId>
+  <version>${beta.version}</version>
```

Comment:
Similar to the previous case: google-cloud/pom.xml currently it links to ${project.version} for google-cloud-translate.